linkml_model/model/schema/meta.yaml

```diff
@@ -1346,6 +1359,22 @@ classes:
     close_mappings:
       - qb:HierarchicalCodeList
 
+  structured_alias:
+    description: >-
+      object that contains meta data about a synonym or alias including where it came from (source)
+      and its scope (narrow, broad, etc.)
+    slots:
+      - name
```

Comment:
ah, unfortunately this is problematic. The metamodel follows a tradition in logic/linked data in using "name" to designate an identifier - and indeed `name` is marked as being an identifier, which will cause problems here, e.g. it will be inlined as a dict with the "names" being keys.

A perfectly acceptable alternate would be `alias` but that is actually slot-specific and used to designate the preferred label for the slot (i.e. unmangled)

Naming things is hard, I might just go with `alias_value` or `label`